### DISCUSSION TITLE
Generic File error

### DISCUSSION THREAD

**Discussion by SimoneTentori** [FeynCalc/feyncalc#275] [General]
Dear all,

I am using FeynCalc 9.3.1.

I am trying to check an amplitude compute with Feynarts+FormCalc using a FeynArts model that I have created using FeynRules.
The model works in the FeynArts+FormCalc setup, unfortunately when I try to create the FA amplitude after 

`$LoadAddOns = {"FeynArts"}`

I obtain the error

**
TagSetDelayed::tagnf: Tag FourVector not found in -Overscript[mom_, _]^mu_.
**

That seems to refer to the first line of .gen model. The error is not clear to me, how could I solve it?

Many thanks, 
Simone 

P.S. I attached the .gen and .mod files
[Modmod.txt](https://github.com/user-attachments/files/15957479/Modmod.txt)
[Modgen.txt](https://github.com/user-attachments/files/15957480/Modgen.txt)

**Comment by vsht** [comment]
Hi,

you need to patch your model before using it with FeynCalc

https://feyncalc.github.io/FeynCalcBook/Extra/CustomModels.html

Cheers,
Vlad

Am 24.06.24 um 16:53 schrieb Simone Tentori:

**Comment by SimoneTentori** [comment]
Dear Vlad,

thank you so much for your answer, this has solved my problem.

Simone
